README.rst

```diff
@@ -5,6 +5,7 @@ USE AT YOUR OWN RISK!
 
 The preferred way is that you put a link to your own repository here:
 
+- https://github.com/aditosoftware/nodebackup (collect data from config files, docker and kubernetes labels and create backups based of this information)
```

Comment:
... based on ...

or just turn it around:

create borg backups based on ...

Reply:
It is now better?